tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -779,13 +779,16 @@ describe('scalar card', () => {
     expect(visibleRunIds).toEqual(['run3']);
   }));
 
-  describe('overflow menu', () => {
-    beforeEach(() => {
+  describe('controls', () => {
+    const ByCss = {
+      FIT_TO_DOMAIN: By.css('[aria-label="Fit line chart domains to data"]'),
+    };
+
+    it('resets domain when user clicks on reset button', fakeAsync(() => {
```

Comment:
Just checking, since this test case doesn't do any `flush()`ing, can `fakeAsync` be dropped?

Reply:
It is used in the `createComponent` or one of these utils (I tried and it threw an error).